spatial_db_msgs/ObjectInstance[] objects
---
int32[] ids
